### DISCUSSION TITLE
[Feature] Two-step verification

### DISCUSSION THREAD

**Discussion by laurenafolk** [immich-app/immich#11937] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Despite OAuth support, two-step verification is more convenient and less difficult to configure. I hope immich web and app can support this feature, thanks.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5478
